README.md

```diff
@@ -1,205 +1,278 @@
+
+### R Code Standards <a name="rinfo"/>
+
+#### Style Guide <a name="rstyle"/>
+
+* __Style guide__: We aim to follow [Hadley Wickham's style guide for R](http://adv-r.had.co.nz/Style.html). This is a work in progress.
+
+To enforce certain elements of this style guide are R repositories now use the [lintr package.](https://github.com/jimhester/lintr) Specifically, we are enforcing *** *** with lintR (these can be found in the .lintr file). Tutorials on how to understand lintr markers or warnings in RStudio and Atom is available in the lintr README.
+
+#### Code Development <a name="rdevel"/>
+
+Code development in our R repositories is complex. To navigate this complexity we use a combination of software tools ([RStudio](https://www.rstudio.com/products/rstudio/download/) and [Atom]()) and testing. This complexity is rooted in our software architecture where many times changes to R code has an effect on another R package and/or the C code. Our R packages revolve around increasing user functionality of our SOILWAT2 C model, so even if code changes only occur in R, you must be conscientious of potential downstream effects.
+
+For example, changes in rSOILWAT2 will almost always require changes to SOILWAT2 (i.e. so that SOILWAT2 can receive a new variable that is being passed from rSOILWAT2) and rSFSW2 (i.e. rSFSW2 would format site-specific information and then use a rSOILWAT2 function to format this information for passage to C).
+
+Here are some guidelines to assist in R code development:
+
+* If your code is a feature, open up a new project on GitHub and sketch out a plan (specific functions) for how you will implement new functionality. Ask for review from your supervisor.
+* Take advantage of [RStudio's debugger](https://support.rstudio.com/hc/en-us/articles/205612627-Debugging-with-RStudio) to walk through functions with similar functionality.
+  - Note: Ironically, sometimes the debugger can be buggy. With patience, you can still walk through functions.
+* Test your new code within an integration project.  
+  - Copy the test project from within the _/Git/_ folder to your _/Desktop_ so that changes to input files aren't integrated onto Git.
+  - Ensure you can run the test project in RStudio prior to any code development. See User Manual (todo) for running a test project.
+  - Change or set options in the descriptions.R file so that your new functionality is triggered (pertains to rSFSW2).
+  - Switching on and off of certain options should NOT be added into _GIT_. If a _new_ option is added though, this should be permanently integrated into the code base (i.e. for rSFSW2 in the _/demo_ folder and the _test project_ folder).
+* Develop code in Atom. To test code, re-install the R package and then re-run the test project within RStudio.
```

Comment:
This is one way to do it. And I would argue a slow one because installing `rSFSW2` takes considerable time. `devtools::load_all` is much quicker and replaces many (but not all) needs.

Reply:
Also, I don't think that re-running a test project again and again (which takes minutes) as I have seen our programmer do it while developing some small new function is an efficient or helpful way to develop code.